src/codeu/chat/server/View.java

```diff
@@ -42,6 +43,7 @@
 
   private final static Logger.Log LOG = Logger.newLog(View.class);
 
+  private static final ServerInfo info = new ServerInfo();
```

Comment:
constants must be uppercase, also info is a  bit vague, SERVER_INFO would be a better choice